storybook/.storybook/preview.ts

```diff
@@ -1,34 +1,47 @@
-import { Preview } from "@storybook/web-components";
+import { GlobalTypes, DecoratorFunction, Parameters } from "@storybook/types";
```

Comment:
Todo:
- bij andere 2 storybooks dezelfde constructie doorvoeren
- Font prefetchen in `<head>` ?